example_config.yaml

```diff
@@ -20,24 +19,22 @@ cognite:
 #Extractor config
 extractor:
     state-store:
-        local:
-            path: state.json
+        raw:
+            database: ${COGNITE_STATE_DB}
+            table: ${COGNITE_STATE_TABLE}
     subscription-batch-size: 10000
     ingest-batch-size: 100000
     poll-time: 5
 
-
 # subscriptions to stream
 subscriptions:
-  - external_id: charts-persisted
-    partitions:
-        - 0
-    lakehouse_abfss_path_dps: abfss://a0fd605a-3a69-401e-a586-bee3bf2c8127@onelake.dfs.fabric.microsoft.com/40b75c4d-5ddd-47ff-bd81-8faadec1804b/Tables/CalculatedTimeseries
-    lakehouse_abfss_path_ts: abfss://a0fd605a-3a69-401e-a586-bee3bf2c8127@onelake.dfs.fabric.microsoft.com/40b75c4d-5ddd-47ff-bd81-8faadec1804b/Tables/CalculatedTimeseriesMetadata
-
-
+    - external_id: ktest_subscription
+      partitions:
+          - 0
+      lakehouse_abfss_path_dps: abfss://a0fd605a-3a69-401e-a586-bee3bf2c8127@onelake.dfs.fabric.microsoft.com/40b75c4d-5ddd-47ff-bd81-8faadec1804b/Tables/CalculatedTimeseries
+      lakehouse_abfss_path_ts: abfss://a0fd605a-3a69-401e-a586-bee3bf2c8127@onelake.dfs.fabric.microsoft.com/40b75c4d-5ddd-47ff-bd81-8faadec1804b/Tables/CalculatedTimeseriesMetadata
```

Comment:
Should these hardcoded paths be put into env variables?

Reply:
its the way they have it

Reply:
I guess it's OK to leave it as is. But we probably need to revisit it when cleaning up the code.